### ISSUE TITLE
Enable 'cpuset' in kernel Kconfig file or mount/enable cgroup controller in your system

### ISSUE THREAD

**Issue by dzintars** [dzintars/infra#28]
```sh
QEMU: Checking for cgroup 'cpuset' controller support                      : WARN (Enable 'cpuset' in kernel Kconfig file or mount/enable cgroup controller in your system)
QEMU: Checking for cgroup 'devices' controller support                     : WARN (Enable 'devices' in kernel Kconfig file or mount/enable cgroup controller in your system)
QEMU: Checking for secure guest support                                    : WARN (Unknown if this platform has Secure Guest support)
LXC: Checking for cgroup 'freezer' controller support                      : FAIL (Enable 'freezer' in kernel Kconfig file or mount/enable cgroup controller in your system)
```

![image](https://user-images.githubusercontent.com/547420/102641591-fc475380-4164-11eb-863d-d3290b41a99a.png)

**Comment by dzintars** [comment]
https://forum.level1techs.com/t/how-can-i-provide-cgroups-to-a-vm-guest/159943/3

**Comment by dzintars** [comment]
```sh
warn: Failed to find cpuset cgroup, you may need to add \"cgroup_enable=cpuset\" to your linux cmdline (/boot/cmdline.txt on a Raspberry Pi)
warn and fatal: Failed to find memory cgroup, you may need to add \"cgroup_memory=1 cgroup_enable=memory\" to your linux cmdline (/boot/cmdline.txt on a Raspberry Pi)
```

**Comment by dzintars** [comment]
`/etc/sysconfig/grub`

```sh
## Example row with Fedora 33 BTRFS ##
GRUB_CMDLINE_LINUX="rhgb quiet cgroup_enable=cpuset"
```

Update grub2 conf
```sh
## BIOS ##
grub2-mkconfig -o /boot/grub2/grub.cfg

## UEFI ##
grub2-mkconfig -o /boot/efi/EFI/fedora/grub.cfg
```

**Comment by dzintars** [comment]
>The validation tool will report the following warning message if the server has Intel processors. It is expected because the validation tool does not check Secure Guest on Intel processors:
```sh
QEMU: Checking for secure guest support: WARN (Unknown if this platform has Secure Guest support)
```
[Source](https://medium.com/@yping88/use-ubuntu-server-20-04-cloud-image-to-create-a-kvm-virtual-machine-with-fixed-network-properties-62ecae025f6c)

**Comment by dzintars** [comment]
The solution of switching to `cgroup v1` is totally WRONG!
Turns out this was the issue of `virt-host-validate` tool and there are the fix in the upstream.
https://gitlab.com/libvirt/libvirt/-/issues/94
